icu4c/source/common/dictbe.cpp

```diff
@@ -1061,8 +1061,10 @@ CjkBreakEngine::CjkBreakEngine(DictionaryMatcher *adoptDictionary, LanguageType
     // Korean dictionary only includes Hangul syllables
     fHangulWordSet.applyPattern(UnicodeString(u"[\\uac00-\\ud7a3]"), status);
     fHangulWordSet.compact();
-    fNumberOrOpenPunctuationSet.applyPattern(UnicodeString(u"[[:Nd:][:Pi:][:Ps:]]"), status);
-    fNumberOrOpenPunctuationSet.compact();
+    // Add number, open punctuation and a-z/A-Z with half width and full width.
+    fNumberOrOpenPunctuationOrLatinSet.applyPattern(
+        UnicodeString(u"[[:Nd:][:Pi:][:Ps:][:Lu:][:Ll:]]"), status);
```

Comment:
Lu+Ll seem broad but incomplete. Should we use Alphabetic instead?
https://www.unicode.org/reports/tr44/#Alphabetic
https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=%5B%3AAlphabetic%3A%5D&abb=on&g=&i=
```suggestion
    // Digits, open punctuation, and "Alphabetic" characters.
    fNumberOrOpenPunctuationOrAlphaSet.applyPattern(
        UnicodeString(u"[[:Nd:][:Pi:][:Ps:][:Alphabetic:]]"), status);
```

Reply:
Thanks. Resolved